lldb/test/API/lang/swift/expression/generic_function_name/TestSwiftGenericFunctionName.py

```diff
@@ -25,7 +25,7 @@ def test(self):
         stream = lldb.SBStream()
         bkpt.GetLocationAtIndex(0).GetDescription(stream, 1)
         desc = stream.GetData()
-        self.assertIn("C.f<T>(T, U) -> ()", desc)
+        self.assertIn("C.f<T>(_:_:)", desc)
```

Comment:
@adrian-prantl the switch from `GetName` to `GetDisplayName` results in this change.